01-git/classworks/Gavruk_Alexander/file.txt

```diff
@@ -0,0 +1 @@
+Gavruk Alexander Vladimirovich
```

Comment:
this file should contain link to the repo where you were doing Solid Train task

Reply:
fixed)